config/environments/test.rb

```diff
@@ -73,4 +75,7 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  # Raise error when a before_action's only/except options reference missing actions
+  config.action_controller.raise_on_missing_callback_actions = false
```

Comment:
By default this was set to true but I have changed it to false as it was failing specs verify_authorized action